skimage/data/__init__.py

```diff
@@ -154,8 +155,30 @@ def rough_wall():
         Some rough wall.
 
     """
-    return load("rough-wall.png")
+    from warnings import warn
+    warn("The rough_wall dataset has been removed due to licensing concerns."
+         "It has been replaced with the gravel dataset. This warning message"
+         "will be replaced with an error in scikit-image 0.17.", stacklevel=2)
+    return gravel()
 
+def gravel():
+    """Gravel
+
+    The original image was downloaded from
+    `CC0Textures <https://cc0textures.com/view.php?tex=Gravel04>`__ and
+    licensed under the Creative Commons CC0 License.
+
+    The downloaded image was then imported to GIMP and converted from color
```

Comment:
Rescaled using which interpolation?  Funny that you did this in GIMP, but maybe it was to get Lanczos which, very sadly, we still don't.

Reply:
Yeah. The look of the image was more important than the actual science in these cases

Reply:
Finally, the big advantage of using GIMP was that I wanted to make sure the images "looked" good.

Reply:
My concern remains the same: why include a dataset that we cannot reproduce? Why not write a small piece of Python to apply all the transformations and store that somewhere? I understand that you just wanted some good test data (success, there!), but I feel we should be able to retrace our steps.

Reply:
Hmm, ok. I'll see what I can do there. I didn't understand you wanted reproducibility.

I'll see what I can do.

Reply:
Have we decided that jpeg give reproducible images? Most of the images are stored as jpeg, so even loading them will yield different results.

I get that we strive for reproducibility, but we aren't using these images to make definitive scientific conclusions. GIMP is the right tool for making a "nice image". scikit-image is a good tool for science. 

The goal of this data is to create a "nice image that doesn't violate copyright and that isn't too large". I think it is better to focus our attention span elsewhere.